internal/pkg/fakeroot/fakeroot_test.go

```diff
@@ -218,9 +218,9 @@ root:100000:65536
 func getUserFn(username string) (*user.User, error) {
 	var prefix string
 
-	splitted := strings.Split(username, "_")
-	prefix = splitted[0]
-	uid, err := strconv.Atoi(splitted[1])
+	split := strings.Split(username, "_")
+	prefix = split[0]
+	uid, err := strconv.Atoi(split[1])
```

Comment:
I would rather leave the archaic past tense form `splitted` because I think `split` can be confused with the function name (and also the verb).  So please change that back here and in all the other cases below.  I don't care about the case of `splittedPort` below; that could either stay or be changed to `splitPort` because it is not ambiguous.

Reply:
I have reverted the `splitted` fix.